micro-rdk-server/sdkconfig.defaults

```diff
@@ -14,10 +14,8 @@ CONFIG_ESP32_DEFAULT_CPU_FREQ_240=y
 CONFIG_ESPTOOLPY_FLASHFREQ_80M=y
 CONFIG_ESPTOOLPY_FLASHMODE_DIO=y
 
-
 CONFIG_MBEDTLS_CERTIFICATE_BUNDLE=y
-// When moving to IDF5 we should be able to use CONFIG_MBEDTLS_CERTIFICATE_BUNDLE_DEFAULT_CMN only
-CONFIG_MBEDTLS_CERTIFICATE_BUNDLE_DEFAULT_FULL=y
+CONFIG_MBEDTLS_CERTIFICATE_BUNDLE_DEFAULT_CMN=y
```

Comment:
the `CONFIG_MBEDTLS_CERTIFICATE_BUNDLE` is on by default. not sure why we make it explicit